tests/common.py

```
@@ -23,13 +27,17 @@ def start_redis(release=None, moduleEnvs=[]):
     if release:
         default_target = default_target.replace("debug", "release")
     target = os.environ.get("TARGET", default_target)
-    r = Redis(port=port)
+    r = Redis(host=host, port=port)
     try:
         r.ping()
-        client = FalkorDB(port=port)
+        client = FalkorDB(host=host, port=port)
         g = client.select_graph("test")
         return
     except:
```

Ruff (0.15.13)

[error] Do not use bare `except`

(E722)